benchmark/benchmark.py

```diff
@@ -32,7 +32,13 @@ def sample_texts(
     return dataset
 
 
-def benchmark_tokenizer_async(ov_tokenizer: CompiledModel, dataset: List[Tuple[str, str]]) -> Tuple[pd.Series, float]:
+def batch_iter(dataset: Iterable, batch: int = 1):
+    dataset_iter = iter(dataset)
+    while next_batch := list(islice(dataset_iter, batch)):
+        yield next_batch
+
+
+def benchmark_tokenizer_async(ov_tokenizer: CompiledModel, dataset: List[Tuple[str, str]], batch: int = 1) -> Tuple[pd.Series, float]:
```

Comment:
looks unrelated changes to optimizations. May be it is better to do it in separate PR? Let us do it.

Reply:
It is for better optimization testing for other tokenizers too.